sql/core/src/main/scala/org/apache/spark/sql/execution/commands.scala

```diff
@@ -181,3 +183,25 @@ case class DescribeCommand(
       child.output.map(field => Row(field.name, field.dataType.toString, null))
   }
 }
+
+/**
+ * :: DeveloperApi ::
+ */
+@DeveloperApi
+case class DDLDescribeCommand(
+    relation: BaseRelation,
+    override val output: Seq[Attribute],
+    isExtended: Boolean) extends RunnableCommand {
+
+  override def run(sqlContext: SQLContext) = {
+    val rows = new ArrayBuffer[Row]()
+    rows += Row("# col_name", "data_type", "comment")
```

Comment:
`"# col_name", "data_type", "comment"` is the output field name, we'd better not take that as part of the output.